GetWellSoon_All/BackEnd/src/main/java/com/model/Trial.java

```diff
@@ -19,154 +19,355 @@
 	
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
-	private int trialId;
+	private int id;
 	
-	//private Date updatedOn;
+	private Date createdAt;
+	private Date processedAt;
 	
-	private String trialName;
+	private Date firstPosted;
```

Comment:
@12arnavg Column name for this field should be first_posted and you might need to add @Column annotation for each field. Please follow same for all other fields as well.